22-Footer

ByMofm opened the pull request:
Añado el componente Footer, con su respectivo Css y datos.

Commit:
Primer commit

Se realiza el primer commit a la branch 22-Footer

Commit:
Update index.js

Galaeno commented on src/component/Footer.js:
```diff
@@ -0,0 +1,86 @@
+      <hr className={style.Rectangle_top} />
+      <div className={style.footer_navigation}>
+        <p className={style.Noticias}>
+          <a href="/Noticias">Noticias</a>
```
Los links deben estar en ingles. Lo mismo para el ressto

ByMofm replied:
Corregido

Galaeno commented on src/component/Footer.js:
```diff
@@ -0,0 +1,86 @@
+          <a href="/Contacto">Contacto</a>
+        </p>
+      </div>
+      <div className={style.Sample_Logo}>
```
Pide que primero este el logo, luego los links y luego las redes sociales

ByMofm replied:
Corregido

Galaeno commented on src/component/Footer.module.css:
```diff
@@ -0,0 +1,188 @@
+}
+
+/* Rectangle */
+.Rectangle_top {
```
Por qué arranca con mayusucula?

ByMofm replied:
Corregido

Galaeno commented on src/data/data.js:
```diff
@@ -0,0 +1,32 @@
+import logo from "../data/images/LOGO-SOMOS.png";
```
Indica que éstos datos se deben obtener del endpoint de datos publicos. ÉSto se realiza en la tarea 17

ByMofm replied:
Actualizado, como quedamos, cree una ruta y añadí la información al back, y ya estoy obteniendo los datos como pedía el requerimiento.

Commit:
Corrección de los links

Todos los links del footer y su respectivo css se pasaron al ingles.

Commit:
Update Footer

Se corrigieron 3 de los comentarios entre ellos:
-Clases css sin mayusculas
-Nombres en ingles
-Estructura de codigo

Commit:
Update

-Eliminado archivo data
-Obtengo la info del endpoint

Galaeno commented:
@ByMofm tenés conflictos

Commit:
Update Footer.js

-Cambie la ruta del endpoint

Commit:
Update index.js

-Regreso a index a su version original

ByMofm commented:
Corregido un conflicto que se generaba en el index.js

Galaeno commented:
@ByMofm tenés conflictos

Commit:
Package-lock

Se trata de solucionar una vez mas el conflicto en le package-lock por los nuevos marge

Commit:
Update Package-lock

New update for resolve conflict.

Commit:
Merge branch 'main' into 22-Footer
